V1.0/mailin.js

```diff
@@ -3,7 +3,7 @@ var crypto = require('crypto');
 
 // Work in progress
 
-Mailin = rest.service(function(base_url,access_key,secret_key) {
+var Mailin = rest.service(function(base_url,access_key,secret_key) {
```

Comment:
Hello @otaviosoares , we appreciate you changes.
Since making Mailin class scope limited, our sample for ex: https://github.com/mailin-api/mailin-api-node-js/blob/master/V2.0/examples/tutorial.js#L2 will lead to an error `ReferenceError: Mailin is not defined`.
Can you please modify your PR by updating sample as well with `var Mailin = require("../mailin.js");`
Thanks